source/test/default_mode_test/Symbiont.test.cc

```diff
@@ -536,6 +536,68 @@ TEST_CASE("Process", "[default]") {
 
         sym.Delete();
     }
+
+    WHEN("The symbiont is free living and horizontal transmission is true") {
+      config.FREE_LIVING_SYMS(1);
+      config.SYM_HORIZ_TRANS_RES(140.0);
+      config.HORIZ_TRANS(true);
+      double int_val = 0;
+      int points = 0;
+
+      WHEN("Fls required reproduction resources is greater than -1") {
+        int free_sym_repro_res = 70;
+        config.FREE_SYM_REPRO_RES(free_sym_repro_res);
+        emp::Ptr<Symbiont> sym = emp::NewPtr<Symbiont>(random, world, &config, int_val, points);
+        emp::WorldPosition location = emp::WorldPosition(0, 10);
+
+        WHEN("Points is less than fls required repro res") {
```

Comment:
These comments are aiming to be human readable, so please expand this so that someone less familiar with the system would understand what it is describing